instrumentation/opentelemetry-instrumentation-mysql/tests/test_mysql_integration.py

```diff
@@ -117,3 +117,25 @@ def test_uninstrument_connection(self, mock_connect):
 
         spans_list = self.memory_exporter.get_finished_spans()
         self.assertEqual(len(spans_list), 1)
+
+    @mock.patch("opentelemetry.instrumentation.dbapi.wrap_connect")
+    # pylint: disable=unused-argument
+    def test_sqlcommenter_enabled(self, event_mocked):
```

Comment:
The tests failed with a connection error, there is a reason to use @mock.patch("opentelemetry.instrumentation.dbapi.wrap_connect")
instead of @mock.patch("mysql.connector.connect") like in the rest of the tests?